File: Auto-generated-tasks/Difficulty-2/Task-3/Solution.pas
Var
	Kol:=4, i:=0:integer;
	Tabl:Array[0..Kol] of char;
Begin
	for i:= 0 to Kol/2 do begin
		Tabl[i * 2] := 'N';
		Tabl[i * 2 + 1] := 'o';
	end;
	for i:= 0 to Kol do begin
		WriteLn("Tabl[", i, "]=", Tabl[i]);
	end;
End.

File: Auto-generated-tasks/Difficulty-2/Task-3/Solution.f90
integer:: Kol = 4
integer:: i = 0
Сharacter(len=Kol):: Tabl
do i = 0, Kol/2
	Tabl[i * 2] = "N";
	Tabl[i * 2 + 1] = "o";
end do
do i = 0, Kol
	print *, "Tabl[", i, "]=", Tabl[i]
end do